src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
```

Comment:
The interface name `Drawable` is a verb, which is not recommended for class/interface names. Consider using a noun or an adjective that describes the capability, such as `DrawableElement` or `DrawableObject`.